spec/mongoid/indexable_spec.rb

```diff
@@ -423,6 +423,18 @@ def self.hereditary?
       end
     end
 
+    context "when providing a geo haystack index with a bucket_size" do
+
+      let(:message) do
+        'The geoHaystack option is deprecated.'
```

Comment:
I think it should be "geoHaystack index type" and not "option". https://www.mongodb.com/docs/manual/core/geohaystack/ just calls it an "index" which I suppose also is valid if you prefer that.